automation/terraform/modules/o1-testnet/testnet.tf

```diff
@@ -116,6 +132,14 @@ module "kubernetes_testnet" {
     }
   ]
 
+  plain_node_configs = [
+    for i in range(var.plain_node_count) : {
+      name               = "plain-node-${i+1}"
+      # libp2p_secret      = "online-plain-nodes-libp2p-${i + 1}-key"
```

Comment:
Remove this if the plain node chart/setup doesnt support it, uncomment if it does